workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/BuildConfigEditor.cs

```diff
@@ -0,0 +1,811 @@
+                options &= ~BuildOptions.EnableHeadlessMode;
+            }
+            else
+            {
+                options &= ~BuildOptions.Development;
+            }
+
+            return options;
+        }
+
+        private BuildOptions ConfigureAndroid(BuildTargetConfig buildTarget)
+        {
+            var options = buildTarget.Options;
+            if (EditorGUILayout.Toggle("Development", options.HasFlag(BuildOptions.Development)))
+            {
+                options |= BuildOptions.Development;
+            }
+            else
+            {
+                options &= ~BuildOptions.Development;
+            }
+
+            return options;
+        }
+
+        private BuildOptions ConfigureIOS(BuildTargetConfig buildTarget)
+        {
+            var options = buildTarget.Options;
+            if (EditorGUILayout.Toggle("Development", options.HasFlag(BuildOptions.Development)))
+            {
+                options |= BuildOptions.Development;
+            }
+            else
+            {
+                options &= ~BuildOptions.Development;
+            }
+
+            return options;
+        }
+
+        private BuildOptions ConfigureOSX(BuildTargetConfig buildTarget)
+        {
+            var options = buildTarget.Options;
+            if (EditorGUILayout.Toggle("Development", options.HasFlag(BuildOptions.Development)))
+            {
+                options |= BuildOptions.Development;
+            }
+            else
+            {
+                options &= ~BuildOptions.Development;
+            }
+
+            if (EditorGUILayout.Toggle("Server build", options.HasFlag(BuildOptions.EnableHeadlessMode)))
+            {
+                options |= BuildOptions.EnableHeadlessMode;
+            }
+            else
+            {
+                options &= ~BuildOptions.EnableHeadlessMode;
+            }
+
+            return options;
+        }
+
+        private BuildOptions ConfigureWindows(BuildTargetConfig buildTarget)
+        {
+            var options = buildTarget.Options;
+            if (EditorGUILayout.Toggle("Development", options.HasFlag(BuildOptions.Development)))
+            {
+                options |= BuildOptions.Development;
+            }
+            else
+            {
+                options &= ~BuildOptions.Development;
+            }
+
+            if (EditorGUILayout.Toggle("Server build", options.HasFlag(BuildOptions.EnableHeadlessMode)))
+            {
+                options |= BuildOptions.EnableHeadlessMode;
+            }
+            else
+            {
+                options &= ~BuildOptions.EnableHeadlessMode;
+            }
+
+            return options;
+        }
+
+        private void RecordUndo(string action)
+        {
+            EditorUtility.SetDirty(target);
+            Undo.RecordObject(target, action);
+            invalidateCachedContent++;
+        }
+
+        private static bool IsBuildTargetError(BuildTargetConfig t)
```

Comment:
single letter parameter

Reply:
I think this is fine for a tiny function.